wire-ios-data-model/Source/MLS/OneOnOne/OneOnOneResolver.swift

```diff
@@ -107,7 +107,19 @@ public final class OneOnOneResolver: OneOnOneResolverInterface {
                 else {
                     return
                 }
+
+                let selfUser = ZMUser.selfUser(in: context)
+
+                if !conversation.isForcedReadOnly {
+                    if !selfUser.supportedProtocols.contains(.mls) {
+                        conversation.appendMLSMigrationMLSNotSupportedForSelfUser(user: selfUser)
+                    } else if !otherUser.supportedProtocols.contains(.mls) {
+                        conversation.appendMLSMigrationMLSNotSupportedForOtherUser(user: otherUser)
+                    }
```

Comment:
What if both `selfUser` and `otherUser` do not contain `.mls` in supportedProtocols, should we show 2 system messages?